CMakeLists.txt

```diff
@@ -109,8 +109,13 @@ include_directories(
   lib/jest/include
   test/include
 )
+add_definitions(
+-D__STDC_CONSTANT_MACROS
+-D__STDC_LIMIT_MACROS
+)
```

Comment:
They are needed for the clang build. There are some ifdefs in the clang headers that require them. They seem to be already defined when using g++.